lib/index.js

```diff
@@ -30,5 +32,7 @@ module.exports = {
     uploadAEMMultipartFile,
     getResourceHeaders,
     AEMUpload,
-    AEMDownload
+    AEMDownload,
+    BlockUpload,
+    BlockDownload
```

Comment:
@tmathern @pheenomenon FYI, I noticed that tests in `e2e/blocktransfer.test.js` were failing. Adding these as exports fixed the failures.

Reply:
ack, thanks!